### ISSUE TITLE
Image is missing alt text

### ISSUE THREAD

**Issue by discospiff** [shruthiramesh194/IS7024#9]
Image is missing alt attribute.  Put a description of the image in the alt attribute, so that users who cannot see the image will understand what it represents.

https://github.com/shruthiramesh194/IS7024/blob/51cf516a119be8123e339de28d3aeed982d0cac7/CincinnatiFireServices/CincinnatiFireServices/Pages/Search.cshtml#L31
